### PR TITLE
Fixed Card Columns section

### PR DESCRIPTION
**Pull request by Vimal-Raghubir**
- Removed duplicate card section in the second column.
- Put third card section into the details portion of the image like the first 2 columns, shifting the lorem ipsum section down into the correct spot.

#26267 defines the second point listed above. #26586 clearly defines the issues addressed in this PR.

### CODE REVIEW

**Comment by Vimal-Raghubir** [comment]
Hi @XhmikosR , any update on this PR?

**Comment by XhmikosR** [comment]
Sorry I'm very busy with work, please be patient until someone has the time to look into the changes.

File: docs/4.1/components/card.md [hunk 1]
```diff
@@ -637,16 +637,14 @@ Cards can be organized into [Masonry](https://masonry.desandro.com/)-like column
       </footer>
     </blockquote>
   </div>
-  <div class="card text-center">
+  <div class="card">
+    <img class="card-img" data-src="holder.js/100px260/" alt="Card image">
```
Comment (andresgalante) [inline docs/4.1/components/card.md]: This should be `card-img-top`

File: docs/4.1/components/card.md [hunk 2]
```diff
@@ -637,16 +637,14 @@ Cards can be organized into [Masonry](https://masonry.desandro.com/)-like column
...
     <div class="card-body">
       <h5 class="card-title">Card title</h5>
-      <p class="card-text">This card has supporting text below as a natural lead-in to additional content.</p>
+      <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This card has even longer content than the first to show that equal height action.</p>
```
Comment (andresgalante) [inline docs/4.1/components/card.md]: This card is not wider, it's taller and it's not showing equal height

**Review by andresgalante — CHANGES_REQUESTED** [review]
Hi @Vimal-Raghubir thanks for the PR, and sorry for taking us so long to get to review it.

I would like to keep more cards to show the masonry grid. Can we fix the text and keep the extra cards at the bottom of the right and middle column please?

**Comment by andresgalante** [comment]
Hi @Vimal-Raghubir Thanks for the changes, I would still like to keep more cards to show the masonry grid. Can you add them please?

thanks

**Comment by Vimal-Raghubir** [comment]
Hi Andres which cards are you referring to?

**Pull request closed without merging**